wallet/how-to/use-non-evm-networks/starknet-snap/connect-to-starknet.md

```diff
@@ -1,6 +1,74 @@
 ---
-description: Connect
+description: Connect your Starknet account to MetaMask
 sidebar_position: 1
 ---
 
-# Connect
\ No newline at end of file
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Connect your Starknet account to MetaMask
+
+To connect a dapp to MetaMask for StarkNet, you can use `get-starknet`. 
```

Comment:
We should make it clear who the audience is. When reading I assume the audience is 

> A starknet dApp developer that wants to have compatibility with Metamask. 

Allowing metamask users to access the dApp by simply visiting its website with metamask installed and installing the starknet-snap when requested. 

For example AVNU : https://www.avnu.fi. Is compatible with Metamask thanks to the starknet-snap. Users that go to https://app.avnu.fi/ and click connect can choose metamask and will be prompted to install the starknet-snap. 

Behind the scene AVNU uses get-starknet.

Reply:
If the audience is a user, then we can mention get-starknet because it is interesting in term of interoperability but we should make it clear that a user will never have to directly interact with get-starknet. A user simply installs the snaps when requested.